stdlib/public/core/Equatable.swift

```diff
@@ -48,17 +48,28 @@
 /// `Comparable` protocols, which allow more uses of your custom type, such as
 /// constructing sets or sorting the elements of a collection.
 ///
```

Comment:
@allevato Would you mind taking a look at this section and the changes to `Hashable.swift`? Thanks!

Reply:
Looks great! Thanks for updating all of this.